### DISCUSSION TITLE
How can I disable livewire dev tools on production env?

### DISCUSSION THREAD

**Discussion by morganealemany** [livewire/livewire#8225] [1. Help]
I have a laravel 10 projetc that used livewire 3 and when I turn on production, Google Chrome livewire dev tools is still enable so anyone can see livewire components and data.
Is there a way to disable it?

Thanks

**Comment by PhiloNL** [comment]
Hi @morganealemany,

All the data you see in the dev tools will always be visible regardless of the environment. If you want to hide certain information you will need to make sure it's not part of a public property on your component. The data of any public property is stored on the client side and therefore accessible by the user. 

Please see the [security section](https://livewire.laravel.com/docs/properties#security-concerns) for more information on securing your application.
